build.gradle

```diff
@@ -22,17 +23,19 @@ buildscript {
       'runtime': "com.google.auto.value:auto-value-annotations:${versions.autoValue}",
     ],
     'support': [
-      'appCompat': "com.android.support:appcompat-v7:${versions.supportLibrary}",
+      'appCompat': "androidx.appcompat:appcompat:${versions.appcompat}",
 
       'test': [
         'espresso': [
-          'core': "com.android.support.test.espresso:espresso-core:${versions.espresso}",
-          'web': "com.android.support.test.espresso:espresso-web:${versions.espresso}",
-          'intents': "com.android.support.test.espresso:espresso-intents:${versions.espresso}",
+          'core': "androidx.test.espresso:espresso-core:${versions.espresso}",
+          'web': "androidx.test.espresso:espresso-web:${versions.espresso}",
+          'intents': "androidx.test.espresso:espresso-intents:${versions.espresso}",
         ],
-        'runner': "com.android.support.test:runner:${versions.testRules}",
-        'rules': "com.android.support.test:rules:${versions.testRules}",
-        'uiautomator': 'com.android.support.test.uiautomator:uiautomator-v18:2.1.3',
+
+        'runner': "androidx.test:runner:${versions.testRules}",
+        'runnerExt': "androidx.test.ext:junit:${versions.junitExt}",
```

Comment:
What do we need this one for?

Reply:
the Test team, in their infinite wisdom, moved `AndroidJUnit4`; but I'll rename this to `junitExt` because that better matches the actual maven coordinates.